src/index.ts

```diff
@@ -92,14 +93,18 @@ namespace Tina {
 	 * Used to add new processes to the processor.
 	 *
 	 * @param name process name to add.
+	 * @param executionGroup your own executionGroup to run the process on.
 	 * @returns a Process object.
 	 */
-	export function process(name: string): Process {
-		if (Process.processes.has(name)) {
-			return Process.processes.get(name)!;
+	export function process(
+		name: string,
+		executionGroup?: RBXScriptSignal,
```

Comment:
Can execution group be the same as in the ECS? It seems weird to have the same name in two places but supporting different functionality

Reply:
I was thinking on this as well, I was going to discuss this with Luna, if processes can be the same as Systems as in the ECS.